docs/options-pbm.md

```diff
@@ -0,0 +1,41 @@
+# Change Percona Backup for MongoDB configuration
+
+The Operator configures Percona Backup for MongoDB (PBM) with the set of default parameters that ensure its correct operation. However, you may want to fine-tune PBM configuration to meet your specific needs. For example, [adjust the node priority for backups :octicons-link-external-16:](https://docs.percona.com/percona-backup-mongodb/usage/backup-priority.html) to use a specific node. Or configure the [parallel download from the storage for a physical restore :octicons-link-external-16:](https://docs.percona.com/percona-backup-mongodb/usage/restore-physical.html#parallel-data-download).
+
+To change PBM configuration, edit the `deploy/cr.yaml` Custom Resource manifest and specify the options in the `backup.configuration` subsection. 
+
+## Examples
+
+This example shows how to adjust the node priority for backups:
+
+```yaml
+spec:
+  backup:
+  	...
+  	configuration:
+      backupOptions:
+        priority:
+          "localhost:28019": 2.5
```

Comment:
let's put a real world example because `localhost` won't ever work on k8s:

```
          "cluster1-rs0-0.cluster1-rs0.psmdb.svc.cluster.local:27017": 2.5
```

Reply:
We have it in our cr.yaml like this... 
Anyway, updated following your suggestion